AppCreationScripts/apps.json

```diff
@@ -35,7 +35,6 @@
             "appId": "4a1aa1d5-c567-49d0-ad0b-cd957a47f842",
             "tenantId": "common",
             "authorityEndpointHost": "https://login.microsoftonline.com/"
-          }
```

Comment:
Still invalid. Seems like you need to add a closing curly bracket after line #36.

Reply:
Add not delete the one at Ln #38.

Reply:
Thanks @archieag 
fixed
Learning of the day: don't try to fix something from the online UI between 2 meetings :)